server/scripts/seed.ts

```diff
@@ -1,36 +1,21 @@
 import Db from 'server/lib/db';
 import log from 'server/lib/log';
 import User from 'server/models/User.model';
-import { makeUser } from 'server/test/utils';
-import { UserObject } from 'server/types/User';
+import { createManyUsers } from 'server/test/utils';
 
-async function create100Users(): Promise<UserObject[]> {
-  const promises = Array.from({ length: 100 }, makeUser);
-  const userArray = await Promise.all(promises);
-  return userArray;
-}
-
-async function insertUsers(userArray: UserObject[]) {
+export async function seed(): Promise<void> {
   try {
+    const userArray = await createManyUsers(100);
     await Db.sequelize.transaction(async transaction => {
       await User.bulkCreate(userArray, { transaction });
     });
-    log('Successfully inserted 100 users');
+    log(`Created ${userArray.length} users!`);
```

Comment:
Line 12 doesn't need to be dynamic if the function itself isn't dynamic. Also, perhaps we should have moved `insertUsers` to Utils and renamed it to `createUsers(count)` (or similar) so it could be used in tests?

Reply:
I'm not 100% sure what correction you are asking for...

Reply:
i can make it dynamic by adding a parameter?

Reply:
I'm not 100% sure what I was suggesting as I wasn't entirely certain what @timmyichen was suggesting in their review, so I was trying to leave it open for discussion. 😂

I think @timmyichen previously suggested merging the two functions, which is essentially what you did. I'm uncertain what direction he was going with that.

My initial suggestion was to move the function to Utils, give it a name that matches whatever pattern we decide on for the Utils functions (eg. `createUsers`), have it return an array of the inserted users, and export the function. Then `seed` can import and use that function to create its users. That would also allow us to also use `createUsers` (or w/e name it's given) in tests.

Reply:
Re-reading his comments, I believe he was suggesting merging `insertUsers` into `create100Users` and then extracting that function out into Utils... so we agreed on the latter part, it should be made into a reusable function in Utils.

The difference in our suggestions was that I would not have merged them and just moved them both. Either works, since there aren't any current use cases for each, merging is fine. If, in the future, someone needs access to the raw data before the users are inserted, they can always extract it to a separate function themself.